seekh-sdk/src/auth/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface login {
```

Comment:
We can rename these types to be a little more descriptive. e.g. LoginPayload, SignUpPayload